flink-cdc-common/src/main/java/org/apache/flink/cdc/common/pipeline/PipelineOptions.java

```diff
@@ -45,6 +45,12 @@ public class PipelineOptions {
                     .defaultValue(1)
                     .withDescription("Parallelism of the pipeline");
 
+    public static final ConfigOption<RuntimeMode> PIPELINE_RUNTIME_MODE =
```

Comment:
Flink has `execution.runtime-mode` configuration, and I assume the two configuration has same semantics, could we follow Flink's design[1] ?
[1] https://github.com/apache/flink/blob/7c03bc834b642ff40828739961a9a4100b9fb073/flink-core-api/src/main/java/org/apache/flink/api/common/RuntimeExecutionMode.java#L39C5-L39C14

Reply:
Regarding this issue, there have been numerous exchanges of views and modifications with other reviewers before. Now, I want to know the following information more clearly:
1.Should we change to the same parameter definition as that of Flink, with our parameter having a lower priority than Flink's own parameters?
2.Is it feasible to use "snapshot" along with Flink's own parameters for control? In this way, when users enable the batch mode, they need to explicitly specify Flink's `execution.runtime-mode` parameter.